wrapper.sh

```diff
@@ -133,8 +164,41 @@ if [[ $create == true ]]; then
     git submodule init
     git submodule import
  
-    echo "import local submodules"
-    git submodule import --recursive --remote --merge
+    echo "update local submodules"
+    #git submodule update --recursive --merge
+
+    #remove _locale directory (only translate English lessons)
+    for dir in `git submodule |  grep "^+"  | cut -d" " -f2`
+        do
+        if [ -d $dir ]
+            then
+            cd $dir
+            if [ `git remote -v | grep "swc-ja" | wc -l` -ge 1 ]
+                then
+                git remote remove swc-ja
+            fi
+            #reset all repos to remote
+            url=https://github.com/swcarpentry-ja/${dir}.git
+            git remote add swc-ja $url
+             if [[ `git branch -v | grep "master" | wc -l` -ge 1 ]]
+                then
+                git pull master -f
+            fi
+            git remote add swc-ja $url                                          
+            if [[ `git branch -v | grep "gh-pages" | wc -l` -ge 1 ]]
+                then
+                git branch -D gh-pages
+            fi
+            git checkout -b gh-pages  `git rev-list HEAD | tail -n 1`
+            git pull swc-ja gh-pages -f
+            git add -u
+            git commit -m "reset branch"
+            git checkout HEAD
+            rm -rf _locale
```

Comment:
The main thing is we want a git repo of the "gh-pages" branch with the `_locale` directory deleted but not commited (so that we can restore it below)